_fossfund/17-curl.md

```diff
@@ -0,0 +1,10 @@
+---
+title: CURL
+logo: curl.svg
```

Comment:
this [image ](https://camo.githubusercontent.com/f6442d1eb0dd05aa511a36868cb47c82da3cc73b7d4d71ed5a873b7d4c02dd4e/68747470733a2f2f6375726c2e73652f6c6f676f2f6375726c2d6c6f676f2e737667)should be added to 

https://github.com/microsoft/opensource.microsoft.com/tree/1dbc9603ccbda0f41c0535267ca502abe80aa0aa/assets/images/fossfund